DataFormats/Math/test/ExtVec_t.cpp

```diff
@@ -1,25 +1,21 @@
-#include "FWCore/Utilities/interface/GCC11Compatibility.h"
-#if GCC_PREREQUISITE(4,8,0)
-
 #include "DataFormats/Math/interface/ExtVec.h"
 
 #include<cmath>
 #include<vector>
 
 #include<iostream>
 
+#ifdef __AVX__
+#define CMS_USE_AVX
+#endif /* __AVX__ */
 
-// #include <x86intrin.h>
+// clang complains that it is not "standard"
+#define constexpr
```

Comment:
maybe better ```s/constexpr //g``` or fix the specific case where clang complained .
IIUC, it complained about T(x) being a non-constexpr in  ```  constexpr Vec xx = T(3.3) + zero;```